### ISSUE TITLE
Webpack 5 support

### ISSUE THREAD

**Issue by danhtran94** [lingui/js-lingui#1045]
**Describe the bug**
`@lingui/loader` not working with webpack 5

**To Reproduce**
run latest nextjs with following config:
```
module.exports = {
  webpack: (config, { isServer }) => {
    config.module.rules = [
      ...config.module.rules,
      {
        test: /\.po/,
        use: [
          {
            loader: "@lingui/loader",
          },
        ],
      },
    ];

    return config;
  },
  future: {
    webpack5: true,
  },
};
```

**Expected behavior**
Errors:
```
error - ./locales/en/messages.po
Error: Could not retrieve module factory for type LoaderDependency
```

**Additional context**
lingui config:
```
module.exports = {
  runtimeConfigModule: {
    i18n: ["@/libs/lingui/i18n.ts", "i18n"],
    Trans: ["@/libs/lingui/Trans.tsx", "Trans"],
  },
  compileNamespace: "ts",
  locales: ["en", "vi"],
  catalogs: [
    {
      path: "locales/{locale}/messages",
      include: ["pages", "components"],
    },
  ],
  format: "po",
};

```

- jsLingui version `3.8.9`
- Babel version `7.13.15`
- Your Babel config
```
{
  "presets": ["next/babel"],
  "plugins": [["macros", {}, "macros-lingui"]]
}

```

**Comment by semoal** [comment]
Released https://github.com/lingui/js-lingui/releases/tag/v3.8.10, give it a try and tell me if everything works fine :)

**Comment by danhtran94** [comment]
It fixed when update to `v3.8.10`
Thank you for awesome repo :D
